src/main/java/mate/academy/spring/service/RentService.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.spring.service;
+
+import java.util.List;
+import mate.academy.spring.entity.Rent;
+
+public interface RentService {
+    void add(Rent rent);
```

Comment:
```suggestion
    Rent rentBook(User user, Book book);
```
Формуй об'єкт rent в середині цього методу
Зробити аналогічний метод  _returnBook_